Project 3 Lol

```diff
@@ -0,0 +1,350 @@
+ ...
+ - Goo_marriage instance_of process
+ - Gall_marriage instance_of process
+ - Goo_marriage occurs_in 3_years
+ - Gall_marriage occurs_in 3_years
+ - Goofus partner_in Goo_marriage
+ - Gallant partner_in Gall_marriage
+  
+```
+
+4. Using the language of First-Order Logic, represent the following natural language expressions; you are welcome to introduce new terms where needed: 
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  ∃Tues.B(S,A) ∧ Wed.B(S,¬A)
+  - 'There exists a Tuesday with Sally bodying an arm and there is a Wednesday with Sally bodying a not-arm.'
+  
+  (b) Every liver has some cell as part at all times it exists.
+  ∀Liver.C
+  - 'All Livers have a Cell'
+  - I don't know how to represent "at all times it exists" in FOL; I was thinking "all livers do not have a not-cell" but that either doesn't make sense, is redundant, or both.
+  
+  (c) John was a child, then an adult, then a senior. 
+  ∃John.Child ∧ John.Grown ∧ John.Old ∧ John.¬(Child ∧ Grown ∧ Old)
+  - 'There exists John as child and John as grown and John as old, and John not-as (child and grown and old), and none of these variables equal each other.'
+  - I don't know how to represent 'pasthood' ("was" and "then") relations in FOL 
+  ...
+  
+  (d) Goofus and Gallant have been married for three years; for each day of that span, it is true to assert they are married. 
+  - ∃Y ⊓ Y(n = 3) ⊓ ∃M ⊓ ∀{F}.M({F},{x}) ⊓ ∀{L}.M({L},{y})
+  class Y = year
+  relation M = marriage
+  individual F = Goofus
+  individual L = Gallant
+  
+```
+
+5. Using BFO and RO, model the following scenario: the content of an rdf file is represented in two serializations - one in Turtle, one in XML - which are sent from one computer to two distinct computers on the same network.   
+- rdf_stuff instance_of generically dependent continuant
+- Turtle_Serialization instance_of generically dependent continuant
+- XML_Serialization instance_of generically dependent continuant
+...
+- rdf_stuff generically_depends_on Turtle_Serialization
+- rdf_stuff generically_depends_on XML_Serialization
+...
+- Computer1 instance_of object
+- Computer2 instance_of object
+- Computer3 instance_of object
+...
+- Wifi_network instance_of object aggregate (project 3 document)
+- Wifi_network connects Computer1
+- Computer1 connected_to Computer2
+- Computer1 connected_to Computer3
+...
+- Turtle_Serialization generically_depends_on Computer1
+- XML_Serialization generically_depends_on Computer1
+- Turtle_Serialization generically_depends_on Computer2
+- XML_Serialization generically_depends_on Computer2
+- Turtle_Serialization generically_depends_on Computer3
+- XML_Serialization generically_depends_on Computer3
+
+6. Using Protege, place these in the BFO hierarchy where you think they fit best:
+```
+  (a) Bach's Well-Tempered Clavier
+  - Process
+  - the sheet music, along with any physical representation of the song (including recordings), are generically dependent continuants.
+  
```

Comment:
By "physical representation of the song" you mean the actual paper sheets? These look like objects. The information content is indeed a generically dependent continuant.